src/plugins/intel_cpu/src/nodes/interpolate.cpp

```diff
@@ -1875,10 +1930,15 @@ void Interpolate::prepareParams() {
 
     auto buildExecutor = [&](const InterpolateKey& key) -> std::shared_ptr<InterpolateExecutor> {
         std::shared_ptr<InterpolateExecutor> executor;
+        const bool isPlanar = key.nodeAttrs.layout == InterpolateLayoutType::planar;
+        // allow I8 planar format only for avx2 with channels = 1, fusing is not supported
+        const bool allowNearestPlanarI8 = isPlanar && mayiuse(cpu::x64::avx2) && !mayiuse(cpu::x64::avx512_core) &&
+            key.nodeAttrs.inPrc == Precision::I8 && key.nodeAttrs.outPrc == Precision::I8 &&
+            key.srcDims[1] == 1 && key.attr.get_post_ops().len() == 0;
```

Comment:
could you please extend ISA and precisions.

Reply:
Done